### DISCUSSION TITLE
Is the Deployment Event truly only GitHub only?

### DISCUSSION THREAD

**Discussion by Dan6erbond** [woodpecker-ci/woodpecker#4275] [Ask a Question]
I'm evaluating Woodpecker as a replacement for Drone, and we really need a feature similar to "promotions" allowing us to manually deploy our apps after successful pipelines.

Woodpecker supposedly has the `event: deployment` that is "[only available for GitHub](https://woodpecker-ci.org/docs/next/usage/workflow-syntax#event)", however the Woodpecker UI has the option to deploy as well:

![image](https://github.com/user-attachments/assets/9275eb02-a4a3-430b-8acd-3336cdc541e3)

Is this just a simple mistake in the docs/outdated information or is the deploy UI something else entirely? Would be good to know how users can migrate from promotion-based workflows and ideally update the docs with specific information on the deployment event or a blog post at least describing it.

**Comment by lafriks** [comment]
There is a `manual` job type that can be used for that

**Comment by qwerty287** [comment]
Yes, the docs are outdated here. Fixed in #4283
